machine-check/src/swarm.rs

```diff
@@ -175,6 +183,18 @@ fn to_swarm(graph: &Graph) -> super::Graph {
     graph.map(|_, n| n.name.clone(), |_, x| x.clone())
 }
 
+fn all_nodes_reachable(graph: &Graph, initial: NodeId) -> Vec<Error> {
+    let visited = Bfs::new(&graph, initial)
```

Comment:
as I don’t think it matters: why is Bfs better than Dfs here?

Reply:
It isn't better in any way. I chose Bfs out of a reflex
because in the first commit I wrote a self-implemented Bfs.
Now that I reviewed again, should I change to Dfs to not pollute the import list?

Reply:
as I said: it doesn’t matter technically, so perhaps add a comment that it doesn’t matter so that our future selves don’t wonder why this uses whatever particular choice you make (since it very much does matter in the other parts of the file)

Reply:
Ah, I see. Nice catch! Switching to Dfs too